docs/ceph-csi-upgrade.md

```diff
@@ -186,7 +193,7 @@ For each node:
   - The pod deletion causes the pods to be restarted and updated automatically
     on the node.
 
-we have successfully upgraded cephfs csi from v1.2.2 to v2.0.0
+we have successfully upgraded cephfs csi from v2.0.0 to v2.0.1
```

Comment:
Should "v1.2.2 to v2.0.1" not be mentioned as well?

Reply:
this doc is for upgrading from v2.0.0 to v2.0.1

Reply:
updated doc now